src/Exception/EntityNotFoundException.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Linio\Common\Exception;
+
+use Throwable;
+
+class EntityNotFoundException extends ClientException
+{
+    public function __construct(
+        string $token = ExceptionTokens::ENTITY_NOT_FOUND,
+        string $message = ExceptionTokens::ENTITY_NOT_FOUND,
```

Comment:
Message would not be the first parameter as `$token` is required, but additional stuff is not. My question is if I should modify `__toString` to include the `$internalDetail` in the output. Also, is `$internalDetail` a good name, or something else?